iommi/form.py

```diff
@@ -154,10 +154,10 @@ def many_to_many_factory_write_to_instance(field, instance, value):
 _field_factory_by_field_type = {}
 
 
-def register_field_factory(django_field_class, *, shortcut_name=MISSING, factory=MISSING):
+def register_field_factory(django_field_class, *, shortcut_name=MISSING, factory=MISSING, **kwargs):
     assert shortcut_name is not MISSING or factory is not MISSING
     if factory is MISSING:
-        factory = Shortcut(call_target__attribute=shortcut_name)
+        factory = Shortcut(call_target__attribute=shortcut_name, **kwargs)
```

Comment:
We should have an else that asserts `not bool(kwargs)` to not drop anything on the floor.